core/src/test/java/io/confluent/kafka/schemaregistry/utils/TestUtils.java

```diff
@@ -98,82 +97,67 @@ public static void waitUntilTrue(Callable<Boolean> callable, long timeoutMs, Str
     }
   }
 
-  public static int registerSchema(String baseUrl, String schemaString, String subject)
+  public static int registerSchema(RestService restService, String schemaString, String subject)
```

Comment:
I think there's an opportunity here for another small refactoring/improvement.

A few of the methods here in TestUtils are essentially wrappers around RestService methods, but exist solely to reduce the boilerplate of creating a RegisterSchemaRequest object with a given schemaString.

What do you think about "moving" these TestUtils methods to RestService? This could make CachedSchemaRegistry a bit more concise as well.